### ISSUE TITLE
unsupported a jackson DeserializationFeature(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES) 

### ISSUE THREAD

**Issue by yzmyyff** [briandilley/jsonrpc4j#99]
I want it can throw an Exception when _JSON NULL_ matched _PRIMITIVE_.
when I pass my ObjectMapper to JsonRpcServer.
It dose not enable the DeserializationFeature.
code like this:

``` java
  ObjectMapper objectMapper = new ObjectMapper();
  objectMapper.enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES);
  this.jsonRpcServer = new JsonRpcServer(objectMapper, this.userService, UserService.class);
```

**Comment by wuwen5** [comment]
I think it has been fixed in #175, https://github.com/briandilley/jsonrpc4j/commit/d91f35a98561fffb02ad5a99fd7c4d61827ed688#diff-79dad87ef55389551762c327fdf04ed7L499.

release at 1.5.1.

**Comment by mslipper** [comment]
Yes, looks like it. @yzmyyff please reopen if this occurs again.
